### DISCUSSION TITLE
Run tests from outside the test project

### DISCUSSION THREAD

**Discussion by jakubpeleska** [webdriverio/webdriverio#7993] [Q&A]
Hi guys!

I have a project with installed `webdriverIO` and I am trying to run tests of a different project.

I was able to run the test but `webdriverIO` was using **async** browser instead of a **sync** for which is the test written and setuped. It is also good to mention that the test is using typescript.

Does anybody have any experience with this topic?

I will appreciate any answer.
Thank you!

**Comment by christian-bromann** [comment]
@jakubpeleska sorry but your question is not really clear, you need to elaborate so people can understand what you are doing and what you are trying to achieve.

**Reply by jakubpeleska** [reply]
@christian-bromann You are right. I didn't put much effort into this question. I have found a solution to my problem so this discussion is not needed. You can delete it if you wish so. Sorry for wasting your time.
